dbt_common/events/functions.py

```diff
@@ -94,7 +94,7 @@ def msg_to_dict(msg: EventMsg) -> dict:
     msg_dict = dict()
     try:
         msg_dict = MessageToDict(
-            msg.data,
```

Comment:
Why this was not caught by CI in this repo? Can we make sure there are some issue tracking adding tests?

Reply:
Tests added in https://github.com/dbt-labs/dbt-common/pull/51